docs/applications/big-data/big-data-in-the-linode-cloud-streaming-data-processing-with-apache-storm.md

```diff
@@ -0,0 +1,1095 @@
+---
+author:
+    name: Karthik Shiraly
+    email: docs@linode.com
+description: 'Deploy Storm cluster on Linode cloud for real-time analytics on streaming datasets.'
+keywords: 'storm,analytics,big data,zookeeper'
+license: '[CC BY-ND 4.0](http://creativecommons.org/licenses/by-nd/4.0/)'
+published: 'Thursday, March 24th, 2016'
+modified: Thursday, March 24th, 2016
+modified_by:
+    name: Linode
+title: 'Big Data in the Linode Cloud: Streaming Data Processing with Apache Storm'
+contributor:
+    name: Karthik Shiraly
+    link: https://github.com/pathbreak
+external_resources:
+- '[Apache Storm project website](http://storm.apache.org/)'
+- '[Apache Storm documentation](http://storm.apache.org/documentation.html)'
+- '[Storm - Distributed and Fault-Tolerant Real-time Computation](http://www.infoq.com/presentations/Storm-Introduction)'
+---
+
+[Apache Storm](http://storm.apache.org/) is a big data technology that enables software, data, and infrastructure engineers to process high velocity, high volume data in real time and extract useful information. Any project that involves processing high velocity data streams in real time can benefit from it.
+
+[Zookeeper](https://zookeeper.apache.org/) is a critical distributed systems technology that Storm depends on to function correctly. 
+
+Some use cases where Storm is a good solution:
+
+-  Twitter data analytics (for example, trend prediction or sentiment analysis)
+-  Stock market analysis
+-  Analysis of server logs
+-  Internet of Things (IoT) sensor data processing
+
+This guide explains how to create Storm clusters on the Linode cloud using a set of shell scripts that use Linode's Application Programming Interfaces (APIs) to programmatically create and configure large clusters. The deployed architecture will look like this:
+
+![Architecture](/docs/assets/storm-architecture-650px.png)
```

Comment:
Thanks, but that wasn't necessary, since we'll probably just replace the graphics with ones made in-house. If you'd like to roll back that commit, it will make it easier to merge my copy edit PR once it's complete.